pkg/console/config/config.go

```diff
@@ -1,169 +1,136 @@
-package tlsconfig
+package config
 
 import (
 	"crypto/tls"
 	"errors"
 	"fmt"
 	"os"
+	"path/filepath"
+	"strconv"
 	"strings"
 	"sync"
-	"sync/atomic"
 
-	"github.com/fsnotify/fsnotify"
 	ocpconfigv1 "github.com/openshift/api/config/v1"
 	"github.com/openshift/library-go/pkg/crypto"
 	"k8s.io/apimachinery/pkg/util/yaml"
 	"k8s.io/client-go/util/cert"
 	"kubevirt.io/client-go/log"
 
 	"github.com/kubevirt/vm-console-proxy/api/v1alpha1"
+	"github.com/kubevirt/vm-console-proxy/pkg/filewatch"
 )
 
-type Watch interface {
-	Run(done <-chan struct{}) error
+const (
+	SaTokensFilename = "use-service-account-tokens"
+)
```

Comment:
nit: I would drop the parenthesis since it's a single `const`.

Reply:
done